README.rst

```diff
@@ -154,6 +178,549 @@ or (to also show test coverage) ::
+                        "source": "cern"
+                  }
+               ],
+               "email": "test@cern.ch",
+               "role": "Co-Producer"
+            }
+      ],
+      "description": "Description",
+      "date": "2024-11-12",
+      "keywords":[
+         {
+            "name": "keyword",
+            "value": {
+                  "name": "keyword"
+            }
+         },
+         {
+            "name": "keyword2",
+            "value": {
+                  "name": "keyword2"
+            }
+         }
+      ],
+      "related_links":[
+         {
+            "name": "related link",
+            "url": "https://relatedlink"
+         }
+      ]
+   }
+
+**Response:**  
+
+Created video JSON.
+
+
+Step 3: Upload the Video
+------------------------
+
+**Request:**  
+
+``PUT`` ``{{baseURL}}/api/files/{{bucket_id}}/{{video_name}}``
+
+**Headers:**  
+
+- ``content-type: video/mp4``
+- ``Accept: application/json, text/plain, */*``
+- ``Accept-Encoding: gzip, deflate, br, zstd``
+
+**Parameters:**
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - **Name**
+     - **Type**
+     - **Location**
+     - **Description**
+   * - **bucket_id**
+     - string
+     - path
+     - Bucket ID.
+   * - **video_name**
+     - string
+     - path
+     - Name of the video file.
+   * - **file**
+     - object
+     - body
+     - Video file.
+
+
+**Response:**  
+
+Uploaded video JSON.
+
+
+Step 4: Create a Flow
+----------------------
+
+**Request:**  
+
+``POST`` ``/api/flows/``
+
+**Headers:**  
+
+- ``content-type: application/vnd.project.partial+json``
+  
+**Parameters:**
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - **Name**
+     - **Type**
+     - **Location**
+     - **Description**
+   * - **version_id**
+     - string
+     - body
+     - Version ID from the uploaded video response.
+   * - **key**
+     - string
+     - body
+     - Video key from the uploaded video response.
+   * - **bucket_id**
+     - string
+     - body
+     - Bucket ID from the Create Video response.
+   * - **deposit_id**
+     - string
+     - body
+     - Deposit ID from the Create Video response.
+
+**Body:**
+
+.. code-block:: json
+
+    {
+      "version_id": "{{main_file_version_id}}",
+      "key": "{{video_key}}",
+      "bucket_id": "{{bucket_id}}",
+      "deposit_id": "{{video_id}}"
+    }
+
+**Response:**  
+
+Created flow JSON.
+
+
+Step 5: (Optional) Upload Additional File
+------------------------------------------
+
+**Request:**  
+
+``PUT`` ``{{baseURL}}/api/files/{{bucket_id}}/{{additional_file}}``
+
+**Parameters:**
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - **Name**
+     - **Type**
+     - **Location**
+     - **Description**
+   * - **bucket_id**
+     - string
+     - path
+     - ID of the bucket to upload the file.
+   * - **file_name**
+     - string
+     - path
+     - Name of the file.
+   * - **file**
+     - file
+     - body
+     - The file to be uploaded.
+
```

Comment:
I think we should differentiate the `Body parameters` and the `Queary parameters`. So that it's more understandable that in some requests users don't need to provide a body and the query is enough

Reply:
They're differentiated using the `Location` column in the table

Reply:
Ok, I see. Thank you for explaining! Then this is fine by me, I don't know maybe @jrcastro2 has some other opinion (reviewed together)